### PR TITLE
62. Update header and footer colours and skin tones

### LINKED ISSUE #62: Update header and footer colours, and skin tones.
## 📄 Context
That is a one-off change in the navbar and footer that will benefit the work to be done on multiple pages.

**PS: Refer to the parent of this issue for references.*

## 📈 Subtasks

- [x] Make sure the fonts are applied correctly based on the Figma description. i.e.  Where to use each font and what size and weight to be applied.
- [x] Apply colour changes to the `NavBar` space where the Logo, links and wallet button lies. 
- [x] Apply colour changes to the `Footer` space. 

## 🎯 Definition of Done

-   [x] The NavBar and Footer match the design specs.

### PR DESCRIPTION
**Pull request by dandheedge**
Fixes #62

### CODE REVIEW

File: packages/ui/src/components/header/NavBar.tsx [hunk 1]
```diff
@@ -35,20 +36,45 @@ export interface NavLinkProps {
     children: ReactNode;
 }
 
-export const NavLink: FC<NavLinkProps> = ({ href, children }) => (
-    <NextLink href={href} passHref>
```
Comment (brunomenezes) [inline packages/ui/src/components/header/NavBar.tsx:39]: Hi @dandheedge, 
Why are you removing the `next/link`? That means a full refresh of the page, losing the SPA effect, and forcing a reconnect of the wallet for every link in the header you click.

File: apps/rollups/src/components/Layout.tsx [hunk 2]
```diff
@@ -36,6 +36,28 @@ export const footerLinks = [
...
+        label: 'About Us',
+        href: 'https://cartesi.io/about/',
+    },
+    {
+        label: 'Docs',
+        href: 'https://docs.cartesi.io/',
+    },
+];
+
+export const footerGeneral = [
```
Comment (brunomenezes) [inline apps/rollups/src/components/Layout.tsx:50]: @dandheedge, 
The links here are redundant. The Figma design is more of a reference in terms of where to go here.

File: apps/staking/src/components/Layout.tsx [hunk 3]
```diff
@@ -51,22 +51,52 @@ export const headerLinks = [
...
     {
         label: 'FAQ',
         href: 'https://github.com/cartesi/noether/wiki/FAQ',
     },
+    {
+        label: 'User Idea',
+        href: 'https://governance.cartesi.io/',
+    },
+    {
+        label: 'Video Guide',
```
Comment (brunomenezes) [inline apps/staking/src/components/Layout.tsx:86]: @dandheedge, 
I would remove that and keep the `how to run a node`. It is the same link. 

For the `User Idea`, I would replace the copy with `Governance`. The **user idea** was a feature discussed in the past related to Explorer Staking, but we did not proceed with that. If someone wants a feature it should be requested through GitHub for consideration.

**Comment by brunomenezes** [comment]
Also, I want to highlight some problems with the `Connect to Wallet` button not being visible on big and small screens (Gifs below), and the items in the footer are all glued up for the `Resources & Security` instead of a list. Also, on the small screen, the `General` area continues to share horizontal space with the `Contracts` area, ideally, the whole footer would become a single column 

### Desktop 



### Small screen

**Review by brunomenezes — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/header/NavBar.tsx (hunk 1 above)
Comment (dandheedge) [reply packages/ui/src/components/header/NavBar.tsx:39]: @brunomenezes my bad, at first I was thinking about the style for css pseudo classes that easier applied on the `<Link>` component, didn't think through this. Going to fix it

File: apps/rollups/src/components/Layout.tsx (hunk 2 above)
Comment (dandheedge) [reply apps/rollups/src/components/Layout.tsx:50]: @brunomenezes alright. I will remove it from the Footer

File: packages/ui/src/components/header/NavBar.tsx (hunk 1 above)
Comment (brunomenezes) [reply packages/ui/src/components/header/NavBar.tsx:39]: You still have the <Link> component; the NextLink just wraps it, so NextJS takes control over the "default" behaviour and keep the routing as a SPA.

**Review by brunomenezes — APPROVED** [review]
Looks good.

**Pull request merged**